### DISCUSSION TITLE
Working with Texts in Slide Masters Feature Missing in v0.50

### DISCUSSION THREAD

**Discussion by bytehammer2** [ShapeCrawler/ShapeCrawler#639] [Q&A]
I'm revisiting my original request "Working with texts in slide masters" which you graciously implemented in v0.32 on Sep 9, 2022, again.

FYI, I always look forward with anticipation to ShapeCrawler updates, but when I tried to update one of my projects using v0.49 to v0.50, I was surprised and somewhat disappointed to discover that this feature is no longer functional/unimplemented. I also discovered that some of ShapeCrawler's code base has been refactored, e.g., IAutoShape to IShape, etc., which is understandable and was easily updated.

Could you please consider reimplementing this feature as soon as possible?

**Comment by bytehammer2** [comment]
Please Disregard.
The issue that was causing the "Not Implemented" exception when trying to set the text in the placeholder textbox in the slide master. It turns out the textbox's Text Options was set to "Resize shape to fit text" which caused the issue. Once I set the textbox's option to "Do not Autofit" the exception no longer occurred.

**Reply by bytehammer2** [reply]
Curiously the previous version didn't have an issue with this text option being set.

**Reply by AdamShakhabov** [reply]
A "Not Implemented" exception is thrown when you try to update a placeholder located on the Slide Master or on the Slide Layout? I cannot reproduce this issue. Is it possible for you to share a sample presentation?

**Reply by AdamShakhabov** [reply]
I was able to reproduce this issue. It will be fixed by this ticket https://github.com/ShapeCrawler/ShapeCrawler/issues/643.

Thank you for the bug report!
